controllers/message.js

```diff
@@ -0,0 +1,13 @@
+const messageService = require('~/services/message')
+
+const createMessage = async (req, res) => {
```

Comment:
i think it should be called sendMessage, cause i don't see any points of just creating messages without sending them

Reply:
Changed.